index.html

```diff
@@ -326,6 +324,25 @@ <h1 class="intro-text">About<br>
         </ul>
       </div>
     </section>
+
+    <section class="contact-wrapper">
+      <div class="contact-container">
+        <h1 class="intro-text">Contact me</h1>
+        <p>
+          If you have an application you are interested in developing,
+          a feature that you need built or a project that needs
+          coding. I’d love to help with it
+        </p>
+      <form action="https://formspree.io/f/mgedddej" method="post">
+        <ul>
+          <li><input name="Name" id="text" type="text" placeholder="Your name" maxlength="30" required></li>
+          <li><input name="Email" id="email" type="email" placeholder="Your email address" required></li>
+          <li><textarea name="comment" placeholder="Write your message here" maxlength="500" required></textarea></li>
+          <li><button type="submit" value="Submit" class="btn">Get In Touch</button></li>

```

Comment:
- [ ] Kindly make your submit button interactive with CSS pseudo classes like :hover
  you design :point_down: 
![Screenshot from 2022-05-31 15-53-03](https://user-images.githubusercontent.com/66097371/171204696-1c7c477d-f695-4575-8536-6dac45a400b0.png)

Reply:
then add active class or visited or focus, I used hover as an example